app/models/info_request.rb

```diff
@@ -1256,7 +1258,7 @@ def recipient_name_and_email
       # TRANSLATORS: Please don't use double quotes (") in this translation
       # or it will break the site's ability to send emails to authorities!
       _("{{law_used_short}} requests at {{public_body}}",
-        law_used_short: law_used_human(:short),
+        law_used_short: legislation.to_s,
```

Comment:
Do we need to actually call `#to_s` for these? Might just call it by default?

Reply:
Of course - changed in 92367a7705